### PR TITLE
Docs 9964 custom app alerts needs mule 4 update hn

### PR DESCRIPTION
**Pull request by hannanelson**
[27-after-Custom-Alerts _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/5955112/27-after-Custom-Alerts._.MuleSoft.Documentation.pdf)

### CODE REVIEW

**Review by MichelleZMY — DISMISSED** [review]
LGTM!

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 1]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-== Prerequisites
+CloudHub provides the ability to display notifications from your app in the Runtime Manager console.
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: "provides the ability" > "enables you"?

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 2]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
-* An error that requires human intervention occurs, such as a problem with data mapping.
-* You want to create a summary of what occurred inside your application, such as the number of orders processed.
+* Deploy your application to CloudHub.
++
+See xref:deploying-to-cloudhub.adoc[Deploy to CloudHub]
+* For Anypoint Studio 6.x and earlier, enable Maven support for your Mule project.
++
+See xref:6@studio::enabling-maven-support-for-a-studio-project.adoc[Enabling Maven Support for a Studio Project].
++
+All projects in Studio 7.x and later are mavenized by default.
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: Wow, I'll bet Apache is not very happy about its trade name becoming a verb....

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 3]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
+You can also create _custom alerts_ for applications deployed to CloudHub.
+To create custom application alerts, you must add Anypoint Connector for CloudHub (CloudHub Connector)
+to flows in your Mule application.
+CloudHub Connector is available for both Mule 3 and Mule 4 applications.
 
-== Deploy Your Application
+=== Create a Custom Alert
 
-To deploy your application, in the Runtime Manager click *Deploy Application* from the *Applications* menu. For more information, see xref:deploying-to-cloudhub.adoc[Deploy to CloudHub].
+To create a custom alert in Anypoint Studio:
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc:57]: I wonder if the first three steps below aren't actually a separate, prerequisite topic to the actual creation of the alert, beginning with "binding the custom notification to an alert"?

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 4]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
 
-== Trigger Your Notification
+. In Studio, add and configure CloudHub Connector in your Mule project:
++
+--
+.. In the Mule Palette view, click *CloudHub* and then drag the *Create Notification* operation to the application flow for which you want to trigger an alert in the canvas.
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]:
```suggestion
.. In the *Mule Palette* view, click *CloudHub* and then drag the *Create Notification* operation to the application flow for which you want to trigger an alert in the canvas.
```

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 5]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
+.The arrow shows the *Username* field in the *Global Element Properties* window.
+image::alerts-custom-ch-connector.png[Username field in the Global Element Properties window]
++
+The connector can use basic authentication or inherited token authentication.
+.. In the *Create Notification* properties window, enter the CloudHub app name in the *Domain* field and the notification in the *Message* field:
++
+.The arrow shows the *Message* field in the *Create Notification* properties window.
+image::alerts-custom-ch-connector-notification.png[Message field in the Create Notification properties window]
++
+For more information, see
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: GLOBALLY enhance cross-refs by making them specific: For more information <about what?>

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 6]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
++
+For more information, see 
+xref:connectors::cloudhub-connector/1.0/#creating-notifications[Creating a Notification].
+--
+. Deploy the app to CloudHub.
+. In Runtime Manager, bind the custom notification to an alert:
++
+--
+.. Click *Alerts* in the menu on the left. 
+.. Click the blue plus icon (*+*) to create a new alert.
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: Here is where you actually create the alert, per the task title.

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 7]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
+Enter the subject line to include in the notification email. 
+Message::
+Enter the custom message to include in the notification.
+Recipients::
+Enter the email addresses for notification recipients. 
 
-Once your application is running, you can trigger the notification by going to your application's URL, for example: `+http://YOURAPP.cloudhub.io/hello-notification+`. The page displays a `Hello World` message, and automatically creates a new notification in your Runtime Manager console. 
++
+You can use variables in the alert *Subject* and *Message* fields.
+For information, see xref:alerts-on-runtime-manager.adoc#alert-variables[Alert Variables].
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: ditto re. specificity

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 8]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
-. Click *Create Alert*.
+. Open the application URL and path, for example:
 +
-image::custom-alert2.png[]
+`+http://myApp.us-w1.cloudhub.io/store+`
 +
-Trigger your notification again by going to your application's URL, for example: `+http://YOURAPP.cloudhub.io/+`. The addresses you specified receive an email with your notification.
+The page displays an error message and sends a new notification to the Runtime Manager console.
+. In the Runtime Manager *Applications* page, click the red bell icon in the upper-right to display the notification.
+. Click the icon to display the error message.
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: Duplicate steps clicking icon?

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 9]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
++
+.The arrow shows the error in the notifications dropdown.
+image::alerts-custom-rtm-notification-error.png[Error in the notifications dropdown]
 
-This is an example of custom properties defining where to send a notification.
+If the notification is sent after an exception, it includes the
+`exception.message` and `exception.stacktrace` as custom properties of the notification,
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc:151]:
```suggestion
`exception.message` and `exception.stacktrace` values as custom properties of the notification, 
```
...or omit "the"

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 10]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
-</cloudhub:create-notification>
-----
-Then, they can be accessed as normal properties in the message, for example `${originalPayload}`
+=== Send an Error Notification
 
-== Send an Error Notification
+You can use the CloudHub Connector inside an `<error-handler>` component to send notifications when errors happen.
 
-You can use the CloudHub Connector inside a catch exception strategy to send notifications when errors happen. 
+For example, use `<on-error-propagate>` component to send the error message and stop processing the flow:
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]:
```suggestion
For example, use the `<on-error-propagate>` component to send the error message and stop processing the flow:
```

File: modules/ROOT/pages/custom-application-alerts.adoc [hunk 11]
```diff
@@ -10,144 +10,176 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
-. Build and deploy your application, then go to the URL: `+http://YOURAPP.cloudhub.io/create-notification-from-exception+`. 
-. Return to the Runtime Manager console, where a notification pop-up appears in the top-right. 
-. Click the notifications link, and the notification with the stack trace appears in the notifications list.
-+
-image::notifications-exception.png[notifications_exception]
-+
-. To view the whole stack trace and details, click *More...* and the whole message appears in a pop-up window:
 
-image::notifications-stack-trace.png[notifications_stack_trace]
+For information on the On-Error components, see
```
Comment (kbostrom) [inline modules/ROOT/pages/custom-application-alerts.adoc]: ditto specificity
GLOBAL "information on" > "information about," per style ref.

**Review by kbostrom — DISMISSED** [review]
CE per request

File: modules/ROOT/pages/custom-application-alerts.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/custom-application-alerts.adoc]: Seems to be a standard term. A quick google reveals a **-mavenize** option in RedHat.

File: modules/ROOT/pages/custom-application-alerts.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/custom-application-alerts.adoc]: Seems redundant, but OK.
```
For more information about creating a notification with CloudHub Connector, see 
xref:connectors::cloudhub-connector/1.0/#creating-notifications[Creating a Notification].
```

File: modules/ROOT/pages/custom-application-alerts.adoc (hunk 7 above)
Comment (hannanelson) [reply modules/ROOT/pages/custom-application-alerts.adoc]: Ditto redundant, but OK

`For information about using variables in alerts, see xref:alerts-on-runtime-manager.adoc#alert-variables[Alert Variables].
`

**Review by dukesphere — APPROVED** [review]

**Pull request merged**